### PR TITLE
Reading TPG parameters from file

### PR DESCRIPTION
**Pull request by depasse**
Reading TPG parameters from TPG_beamv6_trans_xxx_spikekill.txt file

### CODE REVIEW

**Comment by ggovi** [comment]
+1

File: CondTools/Ecal/interface/EcalTPGFineGrainTowerfromFile.h [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#ifndef ECAL_TPGFINEGRAINTOWER_H
+#define ECAL_TPGFINEGRAINTOWER_H
+
+#include <vector>
```
Comment (davidlange6) [inline CondTools/Ecal/interface/EcalTPGFineGrainTowerfromFile.h]: @depasse - it looks like only a few of these includes are needed in this header. Please remove the unneeded ones. Thanks

Comment (depasse) [inline CondTools/Ecal/interface/EcalTPGFineGrainTowerfromFile.h]: OK.  Done (for both  .h files) and resubmitted

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**